src/reports/path.test.ts

```diff
@@ -1,5 +1,5 @@
 import { expect } from "chai";
-import { path } from "./report";
+import { path } from "./path";
```

Comment:
unsure if `reports/path` makes sense as a file? 🤷

Reply:
> unsure if `reports/path` makes sense as a file? 🤷

Is a bit confusing to me, to be honest. Maybe keep in root path, or somehow incorporate into `models`, since that's where it's used? What else would a `reports` folder contain?

Reply:
I'll move it into models, `models/api-path`?

Reply:
> I'll move it into models, `models/api-path`?

It could work. Kinda muddies the distinction between a proper "model" and utility functions common to those implementations. Maybe a subfolder? Or in an `index.js` ?

A different approach could be to adapt what's implemented here as models as custom hooks, e.g. `useDailyAuthsReportData` and `useDropoffsReportData`, at which point maybe `useReportPath` can be a common hook to compose into those?

Reply:
went with `models/api-path` for now 02593c568a244dfddfb2d0d899e5f13b318d80b5, I didn't have a good insight into the hooks but happy to move this around later